scripts/core/ui/components/utils.ts

```diff
@@ -40,27 +40,32 @@ export const gettext = (text, params = null) => {
 
 /**
  * @ngdoc method
- * @name gettextCatalog
+ * @name gettextPlural
+ * @param {Number} count
  * @param {String} text
+ * @param {String} pluralText
  * @param {Object} params
- * @description Used angular gettext service for displaying localised text on Browser
+ * @description Used angular gettext service for displaying plural localised text on Browser
  */
-export const gettextCatalog = (text, params = null) => {
+const gettextPlural = (count, text, pluralText, params) => {
     if (!text) {
         return '';
     }
 
     const injector = angular.element(document.body).injector();
 
     if (injector) { // in tests this will be empty
-        const translated = injector.get('gettextCatalog').getString(text);
-
-        return params ? injector.get('$interpolate')(translated)(params) : translated;
+        return injector.get('gettextCatalog').getPlural(count, text, pluralText, params);
     }
 
     return text;
 };
 
+export const gettextCatalog = {
+    getString: gettext,
```

Comment:
Why are you exporting `getString`? Aren't simple strings translated with the new helper you recently made in utils?

Reply:
The original gettextCatalog support both gettextCatalog.getPlural and gettextCatalog.getString for text extraction so I added it also on this proxy object.

Reply:
Can you remove it from here? `gettext` already does all we need regarding singular translations. It would only confuse people having two methods doing basically the same thing.